python/lsst/ts/xml/data/sal_interfaces/ESS/ESS_Telemetry.xml

```diff
@@ -1677,4 +1677,102 @@
       <Count>1</Count>
     </item>
   </SALTelemetry>
+  <SALTelemetry>
+    <Subsystem>ESS</Subsystem>
+    <EFDB_Topic>ESS_particleMeasurements</EFDB_Topic>
+    <Description>Particle measurements from the SPS30 Sensirion particle sensor.</Description>
+    <item>
+      <EFDB_Name>sensorName</EFDB_Name>
+      <Description>The user-defined name of the sensor.</Description>
+      <IDL_Type>string</IDL_Type>
+      <IDL_Size>20</IDL_Size>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>timestamp</EFDB_Name>
+      <Description>Time at which the data was read (TAI unix seconds).</Description>
+      <IDL_Type>double</IDL_Type>
+      <Units>second</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>matterConcentration1micron</EFDB_Name>
```

Comment:
Why don't you make `matterConcentration` an array and add a pair attribute called `matterConcentrationSize` then you can publish concentration and size as arrays?

Having the names of the particle sizes in the attribute name is a bit weird, e.g. `matterConcentration2point5micron`